src/vm/jithelpers.cpp

```diff
@@ -3028,6 +3028,106 @@ HCIMPLEND
 //
 //========================================================================
 
+//*************************************************************
+// Array allocation fast path for arrays of value type elements
+//
+
```

Comment:
Could you please add `#include <optsmallperfcritical.h>` / `#include <optdefault.h>` around the impl to match what we do elsewhere for the fast paths.